### DISCUSSION TITLE
Can not dock. The destination station spawned in a too narrow passage between the rocks

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13978] [Bug reports]
ы### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The destination station spawned in a too narrow passage between the rocks

[20240519-camel.zip](https://github.com/FakeFishGames/Barotrauma/files/15370729/20240519-camel.zip)
[20240519-camel-2.zip](https://github.com/FakeFishGames/Barotrauma/files/15370866/20240519-camel-2.zip)

In previous versions of the game I never encountered this error.
Something has been changed in the map generator. Something important.

### Reproduction steps

1. Unpack and load any save file
2. Try to reach destination station and you can not dock.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by KweezyCode** [comment]
![image](https://github.com/FakeFishGames/Barotrauma/assets/36813920/415e6e1a-6a79-45bc-98b3-9f694c799a4e)

same. any fix?

**Reply by Regalis11** [reply]
We haven't yet been able to diagnose the issue, but for the time being there's one potential workaround: you can also enter the outpost by leaving the sub and entering it through the airlock. Sorry for the inconvenience!

**Reply by pavel4win** [reply]
If I enter the outpost by leaving the sub and entering it through the airlock, the delivery mission will fail.

**Comment by DougStep** [comment]
https://www.reddit.com/r/Barotrauma/comments/1f64lt7/natural_terrain_blocking_docking_port/
https://www.reddit.com/r/Barotrauma/comments/18wwasp/no_way_to_dock_on_station/
https://steamcommunity.com/app/602960/discussions/1/3160958175980529725/


![Barotrauma terrain bug 2025-03-08 172031](https://github.com/user-attachments/assets/d4808103-839a-42ba-8b34-aa17530d1e98)
I additionally have run into this on my run but with a beacon station, see attached image.

**Comment by DougStep** [comment]
[temp - Copy.zip](https://github.com/user-attachments/files/19141943/temp.-.Copy.zip)
[Data - Copy.zip](https://github.com/user-attachments/files/19141951/Data.-.Copy.zip)
Not sure if this includes the save but figured maybe it does and helps.

**Comment by Regalis11** [comment]
Thank you for the report! We finally took a proper look at this, now that turrets are being added to outposts and this is becoming an issue with more severe consequences. [Addressed in our private repo now](https://github.com/FakeFishGames/Barotrauma-development/pull/8619), fix coming up in the next unstable build.
